src/corehost/src/main.cpp

```diff
@@ -8,30 +8,53 @@
 #include "utils.h"
 #include "coreclr.h"
 
-void get_tpafile_path(const pal::string_t& app_base, const pal::string_t& app_name, pal::string_t& tpapath)
+void get_tpafile_path(const pal::string_t& app_base, const pal::string_t& app_name, pal::string_t* tpapath)
```

Comment:
:+1: "tpafile" was just an initial name. Now that "deps file" is a concept, let's rename it.